sunpy/map/mapbase.py

```diff
@@ -1491,12 +1491,12 @@ def _pv_values(self):
         """
         Return any PV values in the metadata.
         """
-        pattern = re.compile('pv[0-9]_[0-9]a', re.IGNORECASE)
+        pattern = re.compile('pv[0-9]_[0-9]{1,2}$', re.IGNORECASE)
```

Comment:
Maybe I am reading the spec wrong, but there's no technical reason why you couldn't have i>9? right? if you longitude axis was the 10th in your array?

Reply:
I guess that's insanely unlikely to be the case if you are opening it with Map :laughing:

Reply:
You are correct: `i` can range from 1 to 99

Reply:
Now fixed by #7985